ksqldb-common/src/main/java/io/confluent/ksql/properties/with/CommonCreateConfigs.java

```diff
@@ -123,6 +124,16 @@ public static void addToConfigDef(
             Importance.LOW,
             "The fully qualified name of the Avro schema to use"
         )
+        .define(
+            KEY_DELIMITER_PROPERTY,
+            ConfigDef.Type.STRING,
+            null,
+            ConfigValidators.nullsAllowed(ConfigValidators.parses(Delimiter::parse)),
+            Importance.LOW,
+            "The delimiter to use when KEY_FORMAT='DELIMITED'. Supports single "
+              + "character to be a delimiter, defaults to ','. For space and tab delimited values "
+              + "you must use the special values 'SPACE' or 'TAB', not an actual space or tab "
+              + "character. Also see " + VALUE_DELIMITER_PROPERTY)
```

Comment:
Out of curiosity, is it standard (in our codebase, and in general) to cross-link between key and value properties like this? I haven't seen it done elsewhere, and this PR also doesn't add a link from `VALUE_DELIMITER_PROPERTY` to `KEY_DELIMITER_PROPERTY`. Curious what made you decide to link in this direction in this particular case.

Reply:
I just thought that with the format properties it might be a little confusing if you specify `FORMAT='DELIMITED'` and then have to specify individually the format properties (e.g. if you set just `KEY_DELIMITER` you might not get expected behavior). It could be overkill, but I'll just add the cross-link for the VALUE_FORMAT as well.